### PR TITLE
doc: clean out release notes post branch-off

### PR DESCRIPTION
**Pull request by fanquake**
Usually done during a version bump. i.e: https://github.com/bitcoin/bitcoin/commit/d84c9aa25d8b86d7b3ff2205560986f012da727a.

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -60,422 +55,9 @@ From Bitcoin Core 0.20.0 onwards, macOS versions earlier than 10.12 are no
...
-  unloaded in the GUI get removed from the settings list so they won't load
-  again automatically next startup. (#19754)
-
-- The GUI Peers window no longer displays a "Ban Score" field. This is part of
-  changes in 0.20.1 and in this release to the handling of misbehaving
-  peers. Refer to "Changes regarding misbehaving peers" in the 0.20.1 release
-  notes for details. (#19512)
-
-Low-level changes
-=================
```
Comment (MarcoFalke) [inline doc/release-notes.md:451]: I think you can keep the headings. Otherwise they will have to be added back soon

File: doc/release-notes.md [hunk 2]
```diff
@@ -60,422 +55,9 @@ From Bitcoin Core 0.20.0 onwards, macOS versions earlier than 10.12 are no
...
-
-- The `walletcreatefundedpsbt` RPC call will now fail with
-  `Insufficient funds` when inputs are manually selected but are not enough to cover
-  the outputs and fee. Additional inputs can automatically be added through the
-  new `add_inputs` option. (#16377)
-
-- The `fundrawtransaction` RPC now supports `add_inputs` option that when `false`
-  prevents adding more inputs if necessary and consequently the RPC fails.
-
-Changes to Wallet or GUI related RPCs can be found in the GUI or Wallet section below.
```
Comment (MarcoFalke) [inline doc/release-notes.md:171]: This can be kept, too

File: doc/release-notes.md [hunk 3]
```diff
@@ -60,422 +55,9 @@ From Bitcoin Core 0.20.0 onwards, macOS versions earlier than 10.12 are no
...
-  compatibility, `noban` implies the `download` permission, but this may change
-  in future releases. Refer to the help of the affected settings `-whitebind`
-  and `-whitelist` for more details. (#19191)
-
-- Netmasks that contain 1-bits after 0-bits (the 1-bits are not contiguous on
-  the left side, e.g. 255.0.255.255) are no longer accepted. They are invalid
-  according to RFC 4632. Netmasks are used in the `-rpcallowip` and `-whitelist`
-  configuration options and in the `setban` RPC. (#19628)
-
-Changes to Wallet or GUI related settings can be found in the GUI or Wallet  section below.
```
Comment (MarcoFalke) [inline doc/release-notes.md:209]: Same

Comment (fanquake) [reply doc/release-notes.md:209]: done

File: doc/release-notes.md (hunk 2 above)
Comment (fanquake) [reply doc/release-notes.md:171]: done

File: doc/release-notes.md (hunk 1 above)
Comment (fanquake) [reply doc/release-notes.md:451]: Have retained the headings.

**Comment by fanquake** [comment]
Addressed comments and added a commit to bump the gitian descriptors.

**Comment by MarcoFalke** [comment]
review ACK 8a715a6b17a8c10369dcfad735b4362bef0c326c 👜

<details><summary>Show signature and timestamp</summary>

Signature:

```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

review ACK 8a715a6b17a8c10369dcfad735b4362bef0c326c 👜
-----BEGIN PGP SIGNATURE-----

iQGzBAEBCgAdFiEE+rVPoUahrI9sLGYTzit1aX5ppUgFAlwqrYAACgkQzit1aX5p
pUhcqAv/Syxvz5K8yHqx1cjwGRveSprCXDJO1VMmxPkXVsnQpzz5NCNnrMOt3Sut
8GEVLUDU+/ESJPPU9Cy5Vb6b58KvOhPI2JaYYssL6e/KqK2tDGGj/L31/BYfVJjU
i1r3DOAKoi7UVLD1L8HXNFzmp6zvebfsPBIhSz8pOpU2rc/+vBEWj4CfFcLbRpu5
1uzen9jLWhl1GGqN/NuP7ix3E9rr98dmFe1jCjIbV1zr+8RBB1qOxfePQCQaIqgB
T6AbNSKAr2dqRNlfIyQAYMDEI9XcHmib03os9vasaRW4hL70rFu4NVLQYZlBMAlm
qiDUSDOAQEdb5yWF2cCJ1Vy8VSIzAZpBnwu0costrfGGhEGouGgzC91gb6YA7MYo
8ApYA/TimCQjGOgBM3X5ApqEQy5zwOR1eyHmCwn9zJGXBtHwKFacHp7wLJnL2ETt
IBYGF/8/lDYZq467O7u3vmCDyFbkMtZhE8LV21wbpCSckZp3+/hhEXisj8n8S2By
t4OAP9oP
=9HIW
-----END PGP SIGNATURE-----
```

python: error while loading shared libraries: libpython3.8.so.1.0: cannot open shared object file: No such file or directory

</details>

**Comment by laanwj** [comment]
Thanks for doing these already.

**Pull request merged**